### ISSUE TITLE
CFC caching problem with compiled CFC

### ISSUE THREAD

**Issue by adelisle-cegid** [KamasamaK/vscode-cfml#30]
CFC caching is not completing properly because of this error:

```
Detail: File seems to be binary and cannot be opened as text
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4490:262
    at <anonymous>
```

It is pointing to a compiled CFC file from the CF server.

**Comment by KamasamaK** [comment]
Do you have a more full trace? That file is part of core VS Code and unrelated to this extension.

**Comment by adelisle-cegid** [comment]
That's really all I have in the Log (Window) logs:

[2019-01-30 14:02:03.420] [renderer1] [info] no standard startup: panel is active
[2019-01-30 14:02:57.164] [renderer1] [error] cannot open file:///e%3A/workspace/project/server/CFIDE/adminapi/accessmanager.cfc. Detail: File seems to be binary and cannot be opened as text: Error: cannot open file:///e%3A/workspace/project/server/CFIDE/adminapi/accessmanager.cfc. Detail: File seems to be binary and cannot be opened as text
    at file:///C:/Program Files/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:4490:262
    at <anonymous>

I couldn't find a way to exclude my server folder from the caching process, is there one?

**Comment by KamasamaK** [comment]
Thanks for the example. I'm not sure why it's not giving the full trace but now I have an idea of where to look. I wasn't aware that you could have binary CFC files. It looks like they used the `.cfc` extension on a Java class file, which doesn't seem right to me.

Regarding excluding paths, I will look into this. It's possible that one of the core settings like `files.exclude` or `files.watcherExclude` might work for this.

**Comment by adelisle-cegid** [comment]
That's how the CF compile works, it replaces the source file by its compiled bytecode version.

**Comment by KamasamaK** [comment]
This should be fixed with v0.5.3

**Comment by adelisle-cegid** [comment]
Confirmed, no more errors are showing up and everything looks fine.
